mvg/utils/__init__.py

```diff
@@ -0,0 +1,2 @@
+def clamp(value, minimum, maximum):
+    return max(minimum, min(value, maximum))
```

Comment:
A comment would help here and a rearrangement of the parameters

Reply:
I will add docs. I don't get the point in params rearrangement though.